app/__init__.py

```diff
@@ -16,6 +16,19 @@
 s3 = FlaskS3()
 
 
+opensearch_field_name_map = {
```

Comment:
do we not already have some map to take the fields in the FE to the query format? If so, we should combine the two. If not, then we should prtobably utilise this there, right?

Reply:
just had a look through and it seems like it doesn't get rid of other fields, it still maps fields not in this map to some presentation.

Really I think we should be filtering the search to only look for these fields